TribalWars/src/Infrastructure/Repository/BaseRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Infrastructure.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Repository
+{
+    public abstract class BaseRepository
+    {
+        protected readonly ApplicationDbContext _context;
```

Comment:
Most known conventions are separating fields/properties and constructors/methods for better readability. Is it intentional here?